### PR TITLE
Add project details page

### PR DESCRIPTION
**Pull request by khalifan-kfan**
# Description

This is the implementation of the static page for project details at the admin side

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Trello Ticket ID
https://trello.com/c/MZiaRNY9

## How Can This Been Tested?

Pull this branch, login as admin, navigate to projects and test the project details page, it is still static though.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
-  (they have two warnings but on integration with the backend. no warnings will be left)
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots

image
image
image

### CODE REVIEW

**Review by neelxie — APPROVED** [review]
LGTM

File: src/pages/AdminProjectsPage/index.js [hunk 1]
```diff
@@ -126,7 +127,7 @@ const AdminProjectsPage = () => {
                             {contextMenu && project.id === selectedProject && (
                               <div className="BelowHeader bg-light">
                                 <div className="context-menu">
-                                  <div
+                                  {/* <div
```
Comment (e-ian) [inline src/pages/AdminProjectsPage/index.js:130]: @khalifan-kfan let's do away with the commented-out code.

File: src/pages/AdminProjectsPage/index.js [hunk 2]
```diff
@@ -113,6 +113,7 @@ const AdminProjectsPage = () => {
                           <td>{getUserName(project.owner_id)}</td>
                           <td>{project.description}</td>
                           <td>
+                            {/* optional chai */}
```
Comment (e-ian) [inline src/pages/AdminProjectsPage/index.js:116]: ditto

**Review by e-ian — APPROVED** [review]
LGTM - see minor nits in comments for your consideration

**Review by rhodinemma — APPROVED** [review]
Thank you for this.

**Pull request merged**